airflow/migrations/versions/6abdffdd4815_add_dttm_index_on_log_table.py

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""add dttm index on log table
+
+Revision ID: 6abdffdd4815
+Revises: 290244fb8b83
+Create Date: 2023-01-13 13:57:14.412028
+
+"""
+from __future__ import annotations
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "6abdffdd4815"
+down_revision = "290244fb8b83"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Apply add dttm index on log table"""
+    op.create_index("idx_log_dttm", "log", ["dttm"], unique=False)
```

Comment:
You should add it on the following line too:

https://github.com/apache/airflow/blob/911b708ffddd4e7cb6aaeac84048291891eb0f1f/airflow/models/log.py#L42-L45

Reply:
ag, good catch, thanks